staff/tomas-sancheznavarro/app-template/home.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Home page</title>
+</head>
+
+<body>
+    <header>
+        <h1>Welcome To The Coding Bootcamp</h1>
+        <h3>Become a master coder in just 30 days!</h3>
+    </header>
+    <p>Choose your language!</p>
+    <nav>
+        <a href="#">HTML</a> |
+        <a href="#">CSS</a> |
+        <a href="#">JavaScript</a> |
+        <a href="#">Python + Django</a> |
+        <a href="#">Ruby on Rails</a> |
+        <a href="#">PHP</a>
+    </nav>
+    <hr>
+
+    <h2>Our mission...</h2>

```

Comment:
Maybe would be nice to wrap `<h2>Our mission...</h2>` title and text with a `<section>` tag. What do you think?

Further information: 

- [MDN](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines)